stko/optimizers/collapser.py

```diff
@@ -149,70 +152,70 @@ def _get_new_position_matrix(self, mol, step, vectors, scales):
 
         return new_position_matrix
 
-    def _get_BB_vectors(self, mol, BB_atom_ids):
+    def _get_bb_vectors(self, mol, bb_atom_ids):
         """
-        Get the BB to COM vectors.
+        Get the bb to COM vectors.
 
         Parameters
         ----------
         mol : :class:`.Molecule`
             The molecule to be optimized.
 
-        BB_atom_ids : :class:`dict`
+        bb_atom_ids : :class:`dict`
```

Comment:
Maybe for dictionaries it culd be useful to describe the type mapping. i.e :class:`dict` of mapping :class:`int:`::class:`int` so we know exactly what's in them. I believe we do this for lists already. What do you think?

Reply:
Also on this note, I hate how we have to write :class: for every single type, when I think Sphinx can correctly parse arguments types in each docstring correctly without it. How would you feel about filing an issue to change it in the future?

Reply:
Write the issue and add to Notion for the second comment. Top comment fixed.